stock_cycle_count/models/stock_cycle_count.py

```diff
@@ -66,6 +84,17 @@ class StockCycleCount(models.Model):
         readonly=True,
     )
 
+    def write(self, vals):
+        result = super().write(vals)
+        if "responsible_id" in vals and not self.env.context.get("no_propagate"):
+            stock_inventory_records = self.mapped("stock_adjustment_ids")
+            for record in stock_inventory_records:
+                if record.responsible_id.id != vals["responsible_id"]:
+                    record.with_context(no_propagate=True).write(
```

Comment:
I don't really like such code. Don't you have another mean to avoid this ?

Reply:
Why not modifying stock_inventory to change the responsible_id fields to computed field (stored/readonly=False) and override the compute here ?

Reply:
@ArnauCForgeFlow

Reply:
@rousseldenis, we agree with your proposal and have applied it.